zio-http/src/test/scala/zio/http/TestServer.scala

```diff
@@ -137,19 +125,26 @@ final case class TestServer[State](
 }
 
 object TestServer {
+  def addHandler(
+    pf: PartialFunction[Request, ZIO[Any, Nothing, Response]],
+  ): ZIO[TestServer, Nothing, Unit] =
+    ZIO.serviceWithZIO[TestServer](_.addHandler(pf))
 
-  val make: ZIO[Driver with Scope, Throwable, TestServer[Unit]] =
-    make(())
+  def addRequestResponse(
+    request: Request,
+    response: Response,
+  ): ZIO[TestServer, Nothing, Unit] =
+    ZIO.serviceWithZIO[TestServer](_.addRequestResponse(request, response))
 
-  def make[State](initial: State): ZIO[Driver with Scope, Throwable, TestServer[State]] =
+  val layer: ZIO[Driver with Scope, Throwable, TestServer] =
```

Comment:
Is there a way we can return a layer here, rather than a ZIO effect?

Reply:
Yes this should just be `ZLayer.scoped`.

Reply:
Fixed